src/app/spec/test-id-component.directive.ts

```diff
@@ -0,0 +1,10 @@
+import {Directive, Input} from '@angular/core';
+
+@Directive({
+    selector: '[wave-test-id]'
```

Comment:
Do you have ts-lint activated?
![image](https://user-images.githubusercontent.com/4947061/60184445-61e4a900-9828-11e9-9d38-a277e01bb244.png)
Maybe its tedious, but it is helping to spot _real errors_ if our code base is not underlined in red. You can use a project-wide find and replace and the test cases should cover that the replacement works.

Reply:
Yes i have it activated. It stopped complaining at mine when i added wave prefix and brackets, somehow it did not complain about camelCase. I will fix that.